pages/docs/adapters/calculix/adapter-calculix-overview.md

```diff
@@ -23,11 +23,12 @@ Do you Want to build on a cluster? Look at our [instructions for SuperMUC](adapt
 
 ## Versions
 
-Please check the [Calculix adapter README](https://github.com/precice/calculix-adapter/blob/master/README.md) for the newest compatible CalculiX version.
+The latest supported CalculiX version is {{site.calculix_version}}. If you already have a copy of the adapter, check the [adapter README](https://github.com/precice/calculix-adapter/blob/master/README.md) for the CalculiX version it was made for.
 
 Adapters for older versions of CalculiX and preCICE are available in various branches. Branches compatible with **preCICE v2.x:**
 
 * master
+* v2.16
 * v2.15_preCICE2.x
 
```

Comment:
This is seriously outdated. But again, not really relevant to this particular PR.
We should at least add develop and 2.17. But the difference between master and develop is not always consistent...

Reply:
Then let's not treat it in this PR, but in follow-up PRs. Here we should have the main CalculiX version we support consistent across the documentation.

I understand you would add v2.17 and develop here? Then v2.17 should be the global one, right?

Reply:
Let's update in a separate PR.